api/src/client/ApiTaxFilingClient.test.ts

```diff
@@ -96,15 +96,15 @@ const generateSuccessfulApiTaxFilingOnboardingResponse = (
 const generateErroredApiTaxFilingOnboardingResponse = (
   overrides: Partial<ApiTaxFilingOnboardingResponse>,
   statusCode: 400 | 500 | number
-): ApiTaxFilingOnboardingResponse => {
+) => {
   return {
     ApiKey: `some-ApiKey-${randomInt()}`,
     Errors:
       statusCode == 400
         ? [
             {
               Error: `some-error-content-${randomInt()}`,
-              Field: randomInt() % 2 === 0 ? "formFailure" : "businessName",
+              Field: randomInt() % 2 === 0 ? "Taxpayer ID" : "Business Name",
```

Comment:
What's the purpose of this change? I see you've removed the return type on this function now. It appears the type no longer matches "ApiTaxFilingOnboardingResponse" - is that correct? If so, what IS its type? Should this function really be called "generateErroredApiTaxFilingOnboardingResponse" if that's not its return type?